components/gitpod-cli/cmd/tasks-stop.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"syscall"
+	"time"
+
+	supervisor_helper "github.com/gitpod-io/gitpod/gitpod-cli/pkg/supervisor-helper"
+	supervisor "github.com/gitpod-io/gitpod/supervisor/api"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var stopTaskCmdOpts struct {
+	All bool
+}
+
+func stopTask(ctx context.Context, terminalClient supervisor.TerminalServiceClient, terminalAlias string) {
+	terminal, err := terminalClient.Get(context.Background(), &supervisor.GetTerminalRequest{Alias: terminalAlias})
+	if err != nil {
+		panic(err)
+	}
+
+	syscall.Kill(int(terminal.Pid), syscall.SIGINT)
+	syscall.Kill(int(terminal.Pid), syscall.SIGTERM)
+	syscall.Kill(int(terminal.Pid), syscall.SIGKILL)
```

Comment:
we should some time processes to process SIGTERM to finish gracefully, I'm actually not sure we need SIGINT at all

```go
syscall.Kill(int(terminal.Pid), syscall.SIGTERM)
<-time.After(15 * time.Second)
syscall.Kill(int(terminal.Pid), syscall.SIGKILL)
```

Reply:
we should do https://github.com/gitpod-io/gitpod/pull/12116#issuecomment-1214832759 instead